src/pages/ErrorPage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Container, Image, VStack, Center } from '@chakra-ui/react';
+import './ErrorPage.css';
+import image2 from '../assets/images/image2.png';
```

Comment:
we should name the image of the dog something more descriptive, also same for the var name so we can recognize this down the line